src/main/java/at/hannibal2/skyhanni/utils/TimeUtils.kt

```diff
@@ -202,6 +203,12 @@ enum class TimeUnit(val factor: Long, private val shortName: String, private val
     } else shortName
 
     fun format(value: Int, longFormat: Boolean = false) = value.addSeparators() + getName(value, longFormat)
+
+    companion object {
+        fun fromTimeFormatEntryOrNull(entry: CropMilestonesConfig.TimeFormatEntry): TimeUnit? {
```

Comment:
time utils file should be agnostic of any other features or objects. rather move that function into the crop milestone a garden/crop api or create a official enum with those names inside time utils, not the config